### ISSUE TITLE
Client JVM disconnect message can be misleading

### ISSUE THREAD

**Issue by mpirvu** [eclipse-openj9/openj9#16381]
When a client JVM that has just started wants to issue its first compilation request to JITServer, but the server is not available, the client will print a message of the form:
```
#JITServer: t=    10 Lost connection to the server (serverUID=0). Retry immediately: 0.
```
This is misleading because the client never connected to the server (this is the first connection attempt).
It may be better to distinguish between a first connection attempt and subsequent attempts and change the message accordingly.
Related: #16338

**Comment by mpirvu** [comment]
jenkins test sanity xlinuxjit jdk17

**Comment by mpirvu** [comment]
Fixed by #16394
